recipes/hippomocks/all/conanfile.py

```diff
@@ -0,0 +1,40 @@
+from conans import CMake, ConanFile, ConanFile, tools
+import os, glob
+
+class HippomocksConan(ConanFile):
+    name = 'hippomocks'
+    libname = 'HippoMocks'
+    description = 'Single-header mocking framework.'
+    topics = ("conan", "hippomocks", "mock", "framework")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = 'https://github.com/dascandy/hippomocks.git'
+    license = 'LGPL-2.1'
+    no_copy_source = True
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob("%s-*" % (self.name))[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
```

Comment:
The cmake script is only there to build the test.
So I think it can be dropped.

Other header-only recipes do the same.
```suggestion
    no_copy_source = True

    @property
    def _source_subfolder(self):
        return "source_subfolder"

    def source(self):
        tools.get(**self.conan_data["sources"][self.version])
        extracted_dir = glob.glob("%s-*" % (self.name))[0]
        os.rename(extracted_dir, self._source_subfolder)

    def build(self):
        pass

    def package(self):
```